src/redux/categories/categoriesSlice.js

```diff
@@ -43,12 +71,19 @@ export const selectAllCategories = (state) => state.categories.categories;
 
 export const getCategoryStatus = (state) => state.categories.status;
 
-export const getTotalBooksInCategory = (state) => state.categories.totalBooksInCategory;
+export const getTotalBooksInCategory = (state) => state.categories.categories.totalBooksInCategory;
 
 export const selectCategoryById = (state, categoryId) => state.categories.categories.find(
   (category) => category.id === categoryId,
 );
 
-export const { addCategory, removeCategory, updateCategory } = categoriesSlice.actions;
+/* export const getTotalBooksInCategoryById = (state, categoryId) =>
+state.categories.categories[`${categoryId}`].totalBooksInCategory;
+ */
```

Comment:
### Optional
- [ ] It would be nice if we remove all commented piece of code, it makes our codebase looks messy and difficult to understand. Although it is not in the requirement, it is important to remove them if not needed